### PR TITLE
docs: update dead URL.

### PR DESCRIPTION
**Pull request by hxtree**
Note: there are no /parse/examples/ folder for csv-parse.  There is project folder /project/examples/ which features parse examples.  Other sub folders have Example section; but not the csv-parse section.

### CODE REVIEW

**Comment by wdavidw** [comment]
Thank you for the PR. The page is now called [recipes](https://csv.js.org/parse/recipes/)

File: src/md/parse/index.md [hunk 1]
```diff
@@ -38,5 +38,5 @@ For older browsers or older versions of Node, use the modules inside "./lib/es5"
 ## Additional help
 
 For usage and examples, you may refer to
-[example page](/parse/examples/),
+[example page](/project/examples/),
```
Comment (hxtree) [inline src/md/parse/index.md:41]:
```suggestion
[recipes page](/parse/recipes/),
```

**Comment by hxtree** [comment]
@wdavidw changed to recipes thanks.

**Comment by wdavidw** [comment]
Thank you

**Pull request merged**